seqr/views/utils/variant_utils.py

```diff
@@ -58,6 +59,24 @@ def update_project_saved_variant_json(project, family_id=None, user=None):
     return updated_saved_variant_guids
 
 
+def parse_saved_variant_json(variant_json, family):
+    if 'xpos' not in variant_json:
+        variant_json['xpos'] = get_xpos(variant_json['chrom'], variant_json['pos'])
+    xpos = variant_json['xpos']
```

Comment:
yeah, this is copy-paste but no harm in cleaning tup while I'm at it!

Reply:
I'm actually worried there are cases where there is `'xpos'` in the json but no `'chrom'` or `'pos'` and then  `setdefault` will error. Since this was jsut a copy-paste I am going to leave it as-is instead of spending time investigating